fern/docs/pages/welcome/quickstart.mdx

```diff
@@ -56,26 +66,46 @@ types:
       rating:
         type: double
         docs: The rating scale is one to five stars
-        
-service:
-  auth: false
-  base-path: /movies
-  endpoints:
-    getMovie:
-      method: GET
-      path: /{movieId}
-      path-parameters:
-        movieId: MovieId
-      response: Movie
-      errors:
-        - MovieDoesNotExistError
 
 errors:
-  MovieDoesNotExistError:
+  DoesNotExistError:
     status-code: 404
     type: MovieId
 ```
+</CodeBlock>
+
+### Step 3: Generate code
+
+By default, `generators.yml` contains the TypeScript Node.js generator.
+
+<CodeBlock title = "generators.yml">
+```diff
+default-group: local
+groups:
+  local:
+    generators:
+      - name: fernapi/fern-typscript-node-sdk
+          version: 0.8.8
+          output:
+            location: local-file-system
+            path: ../generated/sdks/node
+```
+</CodeBlock>
+
+To invoke the generator, run:
+
+```bash
+fern generate
+```
+
+By default, Fern runs the generator in the cloud and compiles your API definition into files. Once complete, you'll see new files in `/generated/sdks/node`.
```

Comment:
>By default, Fern runs the generator in the cloud and compiles your API definition into files **that are downloaded to your system**. 

Possibly mention that the folder will be one level below the fern folder. It's not obvious to look for it there.